docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+---
+# The Redis Container:
+redis:
+  image: redis:3.0
+  ports:
+    - "6379:6379" # Bind host port 6379 to Redis container's port 6379
+
+# The Web container:
+web: &app_base
+  image: vovimayhem/app-dev:mri-2.2
```

Comment:
The example rails app uses an unreleased version of Rails 5.0 (as it's currently being developed), so you'll not find it on the official Docker library.

I also admit that although there's an official Ruby image, I find this one easier to use, as it does not require any building step besides the `bundle install`. (I remember having to run some sort of building step in the official ruby image)